lib/ansible/modules/cloud/amazon/ecs_tag.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, Michael Pechner <mikey@mikey.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = r'''
+---
+module: ecs_tag
+short_description: create and remove tags on ecs clusters.
+notes:
+    - none
+description:
+    - Creates, removes and lists tags for an ecs_cluster.
+       The resource is referenced by its cluster name.  It is designed to be used with complex args (tags), see the examples.
```

Comment:
```suggestion
     - Resources are referenced by their cluster name.
```

I wouldn't bother with the comment about 'complex args', the parameters and examples do this for you.